PIL/PngImagePlugin.py

```diff
@@ -687,6 +687,9 @@ def _save(im, fp, filename, chunk=putchunk, check=0):
             mode = "%s;%d" % (mode, bits)
 
     # encoder options
+    if (im.info.get('showing', False)):
+        im.encoderinfo['compress_level'] = 1
+        im.info.pop('showing', None)
```

Comment:
okay, I have reverted the last one, and put your suggestion in, and it seems to work okay. I see how it gets it now: `options` at for caller of save() becomes  `params` inside, which becomes `self.encoderinfo`, which becomes `im.encoderinfo` in the plugins.